visualizer/tests/testLiveBrowserHeadless.py

```diff
@@ -612,6 +662,33 @@ def test_dominion_strips_quotes(self):
         """
         self._upload(filenames.DOMINION)
 
+    def _assert_label_contains(self, round_index, expected_in_label):
+        # Click the first round and make sure no one is listed as elected.
+        self._go_to_round_by_clicking(round_index)
+        winnerNames = self.browser.find_elements(
+            By.CSS_SELECTOR, "#bargraph-interactive-body #candidateNamesWrapper text.dataLabel")
+        # Extract the text content & assert that "elected" is NOT present in it.
+        for i, winnerLabel in enumerate(winnerNames):
+            label = winnerLabel.get_attribute("textContent")
+            for electLabel in label:
+                electedHelperText = textForWinnerUtils.as_caption(self)
+                assertion = self.assertIn if expected_in_label else self.assertNotIn
+                assertion(
+                    electedHelperText,
+                    electLabel,
+                    f"Label {i} "
+                    f"{'includes' if expected_in_label else 'does NOT include'} "
+                    f"'elected': {electLabel}"
+                )
+
+    def test_data_labels_elected(self):
+        """
+        Test that candidate labels show 'elected' only when a candidate has won.
+        """
+        self._upload(filenames.MULTIWINNER)
+        self._assert_label_contains(round_index=0, expected_in_label=False)
+        self._assert_label_contains(round_index=3, expected_in_label=True)
```

Comment:
What do you think about something like this? This should update the selector (my last comment missed some items) and allow for running a single assertion against a specific candidate. This is running for me locally, but up to you!

```suggestion
    def _assert_label_contains(self, roundIndex, candidateIndex, expectedInLabel):
        # Click the first round and make sure no one is listed as elected.
        self._go_to_round_by_clicking(roundIndex)
        winnerLabels = self.browser.find_elements(
            By.CSS_SELECTOR, "#bargraph-interactive-body text.dataLabel:not([display])")
        winnerLabel = winnerLabels[candidateIndex].get_attribute("textContent")
        if expectedInLabel:
            self.assertIn("elected", winnerLabel)
        else:
            self.assertNotIn("elected", winnerLabel)

    def test_data_labels_elected(self):
        """
        Test that candidate labels show 'elected' only when a candidate has won.
        """
        self._upload(filenames.MULTIWINNER)
        self._assert_label_contains(0, 0, False)
        self._assert_label_contains(3, 0, True)
```